backend/src/schema/resolvers/users/blockedUsers.spec.js

```diff
@@ -0,0 +1,393 @@
+import { createTestClient } from 'apollo-server-testing'
+import createServer from '../../../server'
+import Factory from '../../../seed/factories'
+import { gql } from '../../../jest/helpers'
+import { neode, getDriver } from '../../../bootstrap/neo4j'
+
+const driver = getDriver()
+const factory = Factory()
+const instance = neode()
+
+let currentUser
+let blockedUser
+let authenticatedUser
+let server
+
+beforeEach(() => {
+  authenticatedUser = undefined
+  ;({ server } = createServer({
```

Comment:
I never saw this notation before.
What does the `;` mean? @roschaefer

Reply:
It was added by eslint. The problem is that if you have e.g.
```js
const { a } = something
```
is valid and no problem but
```js
let a
{ a } = something
```
then babel gets confused. You need to wrap it with parantheses and apparently it's good practice (?) to add a semicolon as delimiter?

Reply:
Ah, okay.